ramls/hold.json

```diff
@@ -30,9 +30,12 @@
       "enum": [
         "Open - Not yet filled",
         "Open - Awaiting pickup",
+        "Open - Awaiting delivery",
         "Open - In transit",
         "Closed - Filled",
-        "Closed - Cancelled"
+        "Closed - Cancelled",
+        "Closed - Unfilled",
+        "Closed - Pickup expired"
```

Comment:
Let's create an API test to demonstrate an issue

Reply:
added